rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc

```diff
@@ -20,7 +20,7 @@ Create a {hcp-title} cluster quickly by using the default options and automatic
 
 [IMPORTANT]
 ====
-Since it is not possible to upgrade or convert existing ROSA clusters to a {hcp} architecture, you must creata a new cluster to use {hcp-title} functionality.
+Since it is not possible to upgrade or convert existing ROSA clusters to a {hcp} architecture, you must create a new cluster to use {hcp-title} functionality.
```

Comment:
I am not sure why, but I am not seeing this change reflected in the preview. https://docs.openshift.com/rosa/rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.html 

Have you done a local build to check for the changes there?

Reply:
Ah thank you for providing the correct link!